### ISSUE TITLE
Silent errors, only reported in console

### ISSUE THREAD

**Issue by TigerTJJ** [RobinLinus/snapdrop#259]
Snapdrop works some days, and doesn't work other days. Unclear whether it's a local issue, or maybe with the snapdrop service.
Today, I checked the console when it wasn't working. 
![image](https://user-images.githubusercontent.com/17763198/108165761-1a77f400-70c1-11eb-855e-48d260f8c1e6.png)
The image shows that the connection failed, but there's no feedback to the users, just a lack of a loading bar. A simple alert saying that the connection failed for some reason would be helpful.

**Comment by hello-smile6** [comment]
I agree. Maybe console.error; console.log capture and automatic uploads would be nice.
